tests/functional/base_pages/another-email.page.js

```diff
@@ -0,0 +1,24 @@
+import QuestionPage from "./question.page";
+
+class EmailConfirmationPage extends QuestionPage {
+  constructor() {
+    super("email-confirmation");
+  }
+
+  title() {
+    return '[data-qa="title"]';
+  }
+
+  emailAddress() {
+    return "#email-address";
+  }
+
+  submit() {
+    return '[data-qa="btn-submit"]';
+  }
+
+  errorPanel() {
+    return `[data-qa=error-body] div.panel__body > ul`;
```

Comment:
We removed all chained selectors in #147, this should be changed to use an `id` or `data-qa` attribute to access the element more directly.